source/includes/fundamentals/code-snippets/auth-credentials-aws-session.rst

```diff
@@ -0,0 +1,11 @@
+.. code-block:: java
```

Comment:
I noticed that all of your code snippets are actually .rst files containing only a code block. Is there a reason you would prefer this to a .java file (even if it isn't a complete program) and a literalinclude? This isn't so much a comment or suggestion as it is just curiosity if I ought to follow this pattern as well for consistency.

Reply:
Yes, I'd like to stick to rst for snippets unless it's a full class file (includes imports, class declaration, etc.). If you have an objection, please let me know!